### ISSUE TITLE
FXIOS-451 ⁃ Accessibility support - Text Increase

### ISSUE THREAD

**Issue by nbhasin2** [mozilla-mobile/firefox-ios#6613]
On iOS, users can go and increase the size of the text. However in our native Firefox iOS app it doesn't happen all over the app. 

Settings.app > Display & Brightness > Text Size > Change the text size away from the default value bigger 

**Expected Result:** Size of all items on photon sheet should have increased and not just close button

**Actual Result:** Only the close button size increased

![image](https://user-images.githubusercontent.com/8919439/81983395-53021300-9601-11ea-9324-487a6c9c4104.png)

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-451)

**Comment by asadotzler** [comment]
Other areas of the UI also fail to resize. The addressbar, password manager, and headings in Settings are also affected.

**Comment by asadotzler** [comment]
From an accessibility standpoint, this is a high severity issue and warrants a P1 label. 

One way to think about this issue is that a user will set the text size to something just large enough to be discernible. So, we can evaluate the usability of our features as if the non-conforming text is indiscernible, or invisible.

For the address bar, a large part of the feature is rendered unusable by this bug. (The autocomplete pop-up is also affected, but only partially.) Also, the address bar seems very central to the browsing experience to me. I think it's the most severe example of this bug.

I think the menu is a little less central to the browsing experience so it's a bit less severe, but it's obviously a front and center feature. Also, the menu can still be operated (through guesses or memorizing the positions of the menu items, for example) though not easily.

The Password Manager is next in severity, I think. I honestly don't know how many people interact with it but I'm sure it's fewer than for the address bar and the Firefox menu. It is, however, pretty much completely broken by this bug. I suspect for most people it's a longer list than the Firefox menu so memorizing the positions of your logins won't necessarily be a great strategy. And guessing doesn't help because the resulting password screen is also completely indiscernible. 

The section headings in Settings is probably the least severe of the manifestations of this bug. They're a nice navigation aid as well as an opportunity for explanatory text so we should fix them, but the settings items themselves are all still mostly discoverable and operable.

**Comment by asadotzler** [comment]
The address bar part of this bug separately. https://github.com/mozilla-mobile/firefox-ios/issues/5033

I also discovered that the Page Actions menu is similarly broken and so are the confirmation prompts you get when using some of the page actions.

**Comment by data-sync-user** [comment]
➤ Ashley Thomas commented:

[~rpappalardo@mozilla.com] would your team be able to help us determine the places where this is an issue and take some screenshots? I can provide that information to UX to come up with appropriate solutions

**Comment by SimonBasca** [comment]
After #8525 and #8532 are fixed we can re check this issue.

**Comment by SimonBasca** [comment]
Verified fix on v34.0 (4879).
